config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   root "users#index"
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do 
+      resources :comments, only: [:create, :new]
+      resources :likes, only: [:create]      
+    end
   end
```

Comment:
- [ ] I think there is a problem in your schema since you named your id with name user_id but it should be autor_id not user_id these are original attribute in the Database so I higly recommend adding a new migration to update the column name from user_id to be author_id to let the user be able to create a new user and a new post and so on. please check this [article](https://stackoverflow.com/questions/1992019/how-can-i-rename-a-database-column-in-a-ruby-on-rails-migration) for more information about renaming the column 🔥✨💯


- Also I faced the same problem as the previous code reviewer I tried to explain the problem that I see 
![image](https://user-images.githubusercontent.com/58553711/205202293-57884c87-2649-4766-ba7b-972c3accf947.png)